pkg/scheduler/scheduler_test.go

```diff
@@ -430,17 +430,17 @@ func TestSchedule(t *testing.T) {
 			},
 			wantScheduled: []string{"eng-beta/new"},
 		},
-		"cannot borrow if cohort was assigned": {
+		"cannot borrow if cohort was assigned and will result in overadmission": {
```

Comment:
The behavior was changed, the test needed to be updated.

Reply:
Yes, but I would prefer to keep the setup the same, just adjust the asserts (the `want` values). Otherwise, it is tricky to tell what are the consequences for the previous setup now.

Reply:
The test was updated to check a similar behavior with the new implementation.

Reply:
I think it would be useful to keep the test with the same setup as before to see if anything changed with the exact scenario,. The updated version of the test could be a new test case. Any particular reason not to include the previous setup?